examples/fabric-admin/commands/fabric-sync/Commands.h

```diff
@@ -29,7 +29,6 @@ void registerCommandsFabricSync(Commands & commands, CredentialIssuerCommands *
         make_unique<FabricSyncAddBridgeCommand>(credsIssuerConfig),
         make_unique<FabricSyncRemoveBridgeCommand>(credsIssuerConfig),
         make_unique<FabricSyncDeviceCommand>(credsIssuerConfig),
-        make_unique<FabricAutoSyncCommand>(credsIssuerConfig),
```

Comment:
nit, why is this getting removed here?

If you do want to remove this why not also removed from `examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h`

Reply:
> nit, why is this getting removed here?
> 
> If you do want to remove this why not also removed from `examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h`

+1 I have the same concern/comment/clarification

Reply:
Auto-sync will cause cyclical syncing at this moment,  but we should be able to solve this with proper algorithm using uniqueID.

I don't see this feature in the test plan, do we want to remove this feature for good? then I will clean up all related pieces of this feature.

Reply:
We can leave the command in and mark the command as use at your own risk, but I don't understand why we are removing it

Reply:
Acked